solutions/project/spoke-project-unclass/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: spoke-unclass
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: "A shared vpc project that implements the Spoke from the Hub and Spoke network design. \nThis package should NOT be deployed within a sandbox landing zone.\n"
```

Comment:
please follow this template for the description : https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/b4f3870eec2e4015eb79f188a6cfbb5f0b84bb53/solutions/project/hub-env/Kptfile#L8